contents/product-engineers/product-health-metrics.md

```diff
@@ -14,25 +14,33 @@ tags:
  - Product metrics
 ---
 
-A product is a complicated system you want to keep healthy. Product health metrics help you understand what parts of your product are working as expected and what parts need interventions to make them healthy.
+A product is a complicated system you want to keep healthy. Product health metrics help measure the well-being of your product and uncover areas in need of changes to get it back to healthy.
 
-In this guide, I'll cover:
+In this guide, we cover:
 
 1. What is a product health metric?
-2. Common product health metrics
+2. Common product health metrics to measure
 3. How to choose the right product health metrics
 
+> **Want to get started fast?** Try our pre-built [product health dashboard template](/templates/health-dashboard) to monitor the most important metrics with one click.
+
 ## What is a product health metric?
```

Comment:
Kind of wondering if we should make the article start with this heading, like we have with some recent ones. Don't feel strongly, though.